components/org.wso2.carbon.identity.api.server.claim.management/org.wso2.carbon.identity.rest.api.server.claim.management.v1/src/main/java/org/wso2/carbon/identity/rest/api/server/claim/management/v1/core/ServerClaimManagementService.java

```diff
@@ -1011,6 +1015,8 @@ private LocalClaimResDTO getLocalClaimResDTO(LocalClaim localClaim) {
             }
         }
 
+        addAttributeProfilesToLocalClaimRes(claimProperties, localClaimResDTO);
```

Comment:
Is the method name is complete?

Reply:
Changed to addAttributeProfilesToLocalClaimResponse